util/Migrator/DbScripts/2021-04-16_01_Two_Guid_Array_Type.sql

```diff
@@ -0,0 +1,11 @@
+-- Create TwoGuidIdArray Type
+IF NOT EXISTS (
+    SELECT *
+FROM sys.types
```

Comment:
Please indent this and the next line , or at least format it similar to other `SELECT` statements throughout the database, but indent the entire `SELECT` block to start under the `IF NOT EXISTS` with the final closing parens back fully left-aligned.